Neos.Flow/Classes/Validation/Validator/RegularExpressionValidator.php

```diff
@@ -24,7 +24,8 @@ class RegularExpressionValidator extends AbstractValidator
      * @var array
      */
     protected $supportedOptions = [
-        'regularExpression' => ['', 'The regular expression to use for validation, used as given', 'string', true]
+        'regularExpression' => ['', 'The regular expression to use for validation, used as given', 'string', true],
+        'validationErrorMessage' => ['', 'The error message to show for validation, if the regular expression validation fails', 'string', false]
```

Comment:
This is only enough to fullfill the needs of the Neos validation system so we do not get an exception here, but in fact this new supported option should impact eg.
https://github.com/neos/flow-development-collection/pull/3133/files#diff-2ecda0a3f7d78fb28fd5cffe20976014c4281e728f4193e505844d380144e9f1R43

otherwise it makes absolutely no sense from the PHP side of things, as nothing happens when I set this option.